### PR TITLE
Fix weirdness in IDD For HeatPump:PlantLoop:EIR:XXX

### PR DESCRIPTION
**Pull request by jmarrec**
Pull request overview
---------------------

Fix weirdness in IDD For HeatPump:PlantLoop:EIR:XXX (probably copy paste issue?)

Found by @joseph-robertson while wrapping the objects in the OS SDK: https://github.com/NREL/OpenStudio/pull/4368/files#r677422116

### Pull Request Author
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [ ] Label the PR with at least one of: Defect, Refactoring, NewFeature, Performance, and/or DoNoPublish
 - [ ] Pull requests that impact EnergyPlus code must also include unit tests to cover enhancement or defect repair
 - [ ] Author should provide a "walkthrough" of relevant code changes using a GitHub code review comment process
 - [ ] If any diffs are expected, author must demonstrate they are justified using plots and descriptions
 - [ ] If changes fix a defect, the fix should be demonstrated in plots and descriptions
 - [ ] If any defect files are updated to a more recent version, upload new versions here or on DevSupport
 - [ ] If IDD requires transition, transition source, rules, ExpandObjects, and IDFs must be updated, and add IDDChange label
 - [ ] If structural output changes, add to output rules file and add OutputChange label
 - [ ] If adding/removing any LaTeX docs or figures, update that document's CMakeLists file dependencies

### Reviewer
This will not be exhaustively relevant to every PR.
 - [ ] Perform a Code Review on GitHub
 - [ ] If branch is behind develop, merge develop and build locally to check for side effects of the merge
 - [ ] If defect, verify by running develop branch and reproducing defect, then running PR and reproducing fix
 - [ ] If feature, test running new feature, try creative ways to break it
 - [ ] CI status: all green or justified
 - [ ] Check that performance is not impacted (CI Linux results include performance check)
 - [ ] Run Unit Test(s) locally
 - [ ] Check any new function arguments for performance impacts
 - [ ] Verify IDF naming conventions and styles, memos and notes and defaults
 - [ ] If new idf included, locally check the err file and other outputs

### CODE REVIEW

File: idd/Energy+.idd.in [hunk 1]
```diff
@@ -71731,7 +71731,8 @@ HeatPump:PlantLoop:EIR:Cooling,
         \note object for this cooling object. The companion is used in
         \note sizing the heat pump as well as to allow checks for unexpected
         \note simultaneous operation of the two objects.
-        \reference PLHPHeatingNames
+        \type object-list
+        \object-list PLHPHeatingNames
```
Comment (jmarrec) [inline idd/Energy+.idd.in:71735]: Cooling references heating

File: idd/Energy+.idd.in [hunk 2]
```diff
@@ -71830,8 +71831,7 @@ HeatPump:PlantLoop:EIR:Heating,
         \note sizing the heat pump as well as to allow checks for unexpected
         \note simultaneous operation of the two objects.
         \type object-list
-        \object-list WWHPCoolingNames
-        \reference PLHPCoolingNames
+        \object-list PLHPCoolingNames
```
Comment (jmarrec) [inline idd/Energy+.idd.in:71834]: And Heating references Cooling

Comment (Myoldmopar) [reply idd/Energy+.idd.in:71834]: So this was an object-list type but didn't actually have an object-list defined.  Nice fix.

File: idd/Energy+.idd.in (hunk 1 above)
Comment (Myoldmopar) [reply idd/Energy+.idd.in:71735]: And this was the wrong type.  Good fix.

**Review by Myoldmopar — APPROVED** [review]

**Comment by Myoldmopar** [comment]
No problem with this tiny fix, and CI is happy with it of course.  Merging.

**Pull request merged**